mixpanel_oauth_to_redshift.go

```diff
@@ -0,0 +1,179 @@
+package main
+
+import (
+	"crypto/md5"
+	"database/sql"
+	"encoding/hex"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"sort"
+	"time"
+
+	"github.com/Clever/pathio"
+	_ "github.com/lib/pq" // Postgres Driver
+	env "github.com/segmentio/go-env"
+)
+
+var (
+	awsAccessKeyId     = env.MustGet("AWS_ACCESS_KEY_ID")
+	awsSecretAccessKey = env.MustGet("AWS_SECRET_ACCESS_KEY")
+	awsRegion          = env.MustGet("AWS_REGION")
+	mixpanelAPIKey     = env.MustGet("MIXPANEL_API_KEY")
+	mixpanelAPISecret  = env.MustGet("MIXPANEL_API_SECRET")
+	redshiftHost       = env.MustGet("AWS_REDSHIFT_HOST")
+	redshiftPort       = env.MustGet("AWS_REDSHIFT_PORT")
+	redshiftDatabase   = env.MustGet("AWS_REDSHIFT_DB_NAME")
+	redshiftUser       = env.MustGet("AWS_REDSHIFT_USER")
+	redshiftPassword   = env.MustGet("AWS_REDSHIFT_PASSWORD")
+	jsonpathsFile      = flag.String("jsonpathsfile", "s3://clever-mixpanel/OAuth.json",
```

Comment:
again, wouldn't include this clever-specific path/usage in a public repo (`s3://clever-mixpanel/OAuth`)